### PR TITLE
Responsiveness and Code refactoring

### PR DESCRIPTION
**Pull request by romabulani**
- Added Navbar for Mobile view
- Styled Profile Page
- Not Found Page
- Refactored code and removed the repetitive code

If you dont have time, you can only review in the components/shared folder
- MobileProfile.jsx
- NotFound.jsx

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
Nice refactoring
Code looks good
PR is ready to merge

File: src/components/authentication/LoginForm.jsx [hunk 1]
```diff
@@ -9,77 +10,83 @@ function LoginForm() {
...
+          >
+            Login as Guest
+          </button>
+          {errorData && (
+            <div className="error-login ">
+              <FontAwesomeIcon
+                icon="circle-exclamation"
+                className="error-icon"
+              />
+              {"    "}
```
Comment (anuj-kumary) [inline src/components/authentication/LoginForm.jsx:78]: Remove this curly braces

File: src/components/profile/profile.css [hunk 2]
```diff
@@ -1,8 +1,33 @@
...
+.profile-column {
+  row-gap: 1rem;
+}
+.profile-details {
+  column-gap: 2rem;
+}
+
+.logout-btn {
+  margin-left: 0;
+  margin-top: 0;
```
Comment (anuj-kumary) [inline src/components/profile/profile.css:26]: You can use shorthand property here

Comment (romabulani) [reply src/components/profile/profile.css:26]: Here, I have to keep the margin top and bottom as it is, thats why not assigning the value again

**Review by anuj-kumary — APPROVED** [review]
UI Looks Good and the code is also very clean
Suggestion:-
Remove unwanted curly braces.

**Comment by romabulani** [comment]
> UI Looks Good and the code is also very clean Suggestion:- Remove unwanted curly braces.

Thanks Anuj. These curly braces are for spaces

**Pull request merged**